docker/ol-web-start.sh

```diff
@@ -1,5 +1,13 @@
 #!/bin/bash
 
+if [[ "$USE_NGINX" ]]; then
+  mkdir /etc/nginx || true
+  rm -f /etc/nginx/nginx.conf || true
+  ln -sf /opt/olsystem/etc/nginx/nginx.conf      /etc/nginx/nginx.conf
+  ln -sf /opt/olsystem/etc/nginx/sites-available /etc/nginx/sites-available
```

Comment:
Ah, this shouldn't be `/opt/olsystem`; that path doesn't exist inside the docker container.

```suggestion
  ln -sf /olsystem/etc/nginx/nginx.conf      /etc/nginx/nginx.conf
  ln -sf /olsystem/etc/nginx/sites-available /etc/nginx/sites-available
```

Reply:
Does that mean that I do not need to clone in olsystem?